ui.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+const dns = require('dns');
+const React = require('react');
+const {useState, useEffect} = require('react');
+const Spinner = require('ink-spinner').default;
+const {Box, Text, Newline, useApp} = require('ink');
+
+const api = require('./api');
+
+const ErrorMessage = ({text}) => (
+	<Box>
+		<Text bold color="red">
+			›
+		</Text>
+
+		<Text dimColor>
+			{text}
+		</Text>
+		<br/>
```

Comment:
Could you replace that with a `<Newline/>` component? See https://github.com/vadimdemedes/ink#newline.